### PR TITLE
Report/app solution

### PR DESCRIPTION
**Pull request by Trivivium**

### CODE REVIEW

File: report/REPORT.md [hunk 1]
```diff
@@ -78,194 +94,217 @@ solution in terms of manual work, but it provided us with invaluable learning
...
-adapters to many different database giving us freedom in chosing our storage
+[Entity Framework Core][prog-3] as it integrates very well with ASPNET Core, and
+has adapters to many different database giving us freedom in chosing our storage
 solution later.
 
+[prog-1]: https://dotnet.microsoft.com/
+[prog-2]: https://dotnet.microsoft.com/apps/aspnet
+[prog-3]: https://docs.microsoft.com/en-us/ef/core/
+
 ### Testing
```
Comment (C0DK) [inline report/REPORT.md:148]: This is covered in my PR, but I'll look at your changes and use those instead, where applicable.

File: report/REPORT.md [hunk 2]
```diff
@@ -78,194 +94,217 @@ solution in terms of manual work, but it provided us with invaluable learning
...
+
 Before starting the refactoring of the existing MiniTwit application we
 considered our options and interests of the group members in relation to the
 programming language we aimed to use. This of course had an impact on our
 choices of web application frameworks available to us.
 
-We ended up using .NET Core 3.1 with C# as it was argued that it was the
+We ended up using [.NET Core][prog-1] with C# as it was argued that it was the
 language that most of the group members would be able to write from the start.
 As mentioned a couple of times in previous sections we wanted to focus less on
```
Comment (C0DK) [inline report/REPORT.md:130]: This is not mentioned previously after the restructuring :P

File: report/REPORT.md [hunk 3]
```diff
@@ -78,194 +94,217 @@ solution in terms of manual work, but it provided us with invaluable learning
...
+chapter with a diagram of all the dependencies introduced and the relations
+between and any external services they rely on.
+
+![](./images/dependency_graph.png)
+
+A noticable omission from the diagram is the technologies related to
+containerization and the technologies supporting that (i.e., the operating
+system). These has been omitted since they're a prerequisite of the entire
+application, but doesn't play a role in the functionality of the application.
+The use of containers due add some constraints on the architecture of the
```
Comment (C0DK) [inline report/REPORT.md:283]: not sure i agree. the current solution isn't completly stateless right? and it is containerized?

It's more the otherway. The current solution not being stateless makes it impossible to scale by adding more containers.. 

Right?

**Comment by C0DK** [comment]
So if we consider the three comments i've made and conclude on those, then i'm fine with this PR :+1:

File: report/REPORT.md (hunk 2 above)
Comment (Trivivium) [reply report/REPORT.md:121]: I'm unsure what you are referring to here?

**Pull request merged**